docs/mp/telemetry.adoc

```diff
@@ -0,0 +1,408 @@
+
+=== Setup Jaeger
+
+For the examples Jaeger will be used for gathering of the tracing information.
+
+.Run Jaeger in a docker container.
+[source, bash]
+----
+docker run -d --name jaeger \
+  -e COLLECTOR_ZIPKIN_HOST_PORT=:9411 \
+  -e COLLECTOR_OTLP_ENABLED=true \
+  -p 6831:6831/udp \
+  -p 6832:6832/udp \
+  -p 5778:5778 \
+  -p 16686:16686 \
+  -p 4317:4317 \
+  -p 4318:4318 \
+  -p 14250:14250 \
+  -p 14268:14268 \
+  -p 14269:14269 \
+  -p 9411:9411 \
+  jaegertracing/all-in-one:1.41
+----
+
+All the tracing information gathered from the examples runs is accessible from the browser in the Jaeger UI under link:http://localhost:16686/[]
+
+=== Enable MicroProfile Telemetry in Helidon Application
+
+Together with Helidon Telemetry dependency, an OpenTelemetry Exporter dependency should be added to project's pom.xml file.
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.telemetry</groupId>
+    <artifactId>helidon-microprofile-telemetry</artifactId> <1>
+</dependency>
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>  <2>
+</dependency>
+----
+<1> Helidon Telemetry dependency.
+<2> OpenTelemetry Jaeger exporter.
+
+Add these lines to `META-INF/microprofile-config.properties`:
+
+.MicroProfile Telemetry properties
+[source,properties]
+----
+otel.sdk.disabled=false     <1>
+otel.traces.exporter=jaeger <2>
+otel.exporter.name=greeting-service <3>
+----
+<1> Enable MicroProfile Telemetry.
+<2> Set exporter to Jaeger.
+<3> Name of our service.
+
+Here we enable MicroProfile Telemetry, set tracer to "jaeger" and give a name, which will be used to identify our service in the tracer.
+
+[NOTE]
+====
+for our example we use Jaeger managing the tracing data. If you prefer to use Zipkin, please set `otel.traces.exporter` property to "zipkin". To find more information on how to use Zipkin, please proceed to link:https://zipkin.io/[]. Also a corresponding Maven dependency for the exporter should be added:
+----
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-zipkin</artifactId>
+</dependency>
+----
+====
+
+
+=== Tracing at Method Level
+
+Let us create simple services and use `@WithSpan` and `Tracer` to create span and let MicroProfile OpenTelemetry handle them
+
+[source, java]
+----
+@Path("/greet")
+public class GreetResource {
+
+    @GET
+    @WithSpan("default")                <1>
+    public String getDefaultMessage() {
+        return "Hello World";
+    }
+}
+----
+<1> Use of `@WithSpan` with name "default".
+
+Now let's call the Greeting endpoint:
+
+[source,bash]
+----
+curl localhost:8080/greet
+Hello World
+----
+
+After that if we proceed to Jaeger UI at link:http://localhost:16686/[] the expected output is:
```

Comment:
Next, launch the Jaeger UI at <link>. The expected output is:

Reply:
Fixed